alternative/index.html

```diff
@@ -2,11 +2,11 @@
 layout: default
 title: The best mail merge tools for Gmail
 description: The best mail merge tools for Gmail. Discover Mailmeteor, the best alternative to popular mail merge softwares, including YAMM and Gmass.
-canonical: https://mailmeteor.com/alternative
+canonical: https://mailmeteor.com/alternative/
 
 og_title: The best mail merge tools for Gmail
 og_description: The best mail merge tools for Gmail. Discover Mailmeteor, the best alternative to popular mail merge softwares, including YAMM and Gmass.
-og_url: https://mailmeteor.com/alternative
+og_url: https://mailmeteor.com/alternative/
```

Comment:
```suggestion
og_url: https://mailmeteor.com/alternative/

redirect_from: /alternative
```

Comme on change l'URL de cette page, il faut rediriger l'ancienne ?